src/dmd/dimport.d

```diff
@@ -6,12 +6,12 @@
  * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
  * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
  * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/dimport.d, _dimport.d)
+ * Documentation:  https://dlang.org/phobos/ddmd_dimport.html
```

Comment:
currently the link works but since the package was renamed _dmd_ a few days ago this is _dmd_dimport.html_.

Reply:
dmd_dimport.html currently gives "404 Not Found"

Reply:
Yes, the transition just happened a few days ago and 2.078.0 hasn't been released yet. The link will work in [seven days](https://dlang.org/changelog/2.078.0.html).

Reply:
Same thing here: https://github.com/dlang/dmd/blob/master/src/dmd/target.d#L9-L10